framework/src/physics/PhysicsBase.C

```diff
@@ -29,17 +29,17 @@ PhysicsBase::validParams()
   params.addParam<MooseEnum>(
       "transient", transient_options, "Whether the physics is to be solved as a transient");
 
+  params.addParam<bool>("verbose", false, "Flag to facilitate debugging a Physics");
+
+  // Numerical solve parameters
   params.addParam<std::vector<SolverSystemName>>(
       "system_names",
       {"nl0"},
       "Name of the solver system(s) for the variables. If a single name is specified, "
       "that system is used for all solver variables.");
-
-  MooseEnum pc_options("default none", "none");
+  MooseEnum pc_options("default none no_additional", "no_additional");
```

Comment:
i ll progressively remove none from use

Reply:
What does `no_additional` mean?

Reply:
when could you add additional preconditioning? We don't support `-pc_type composite`

Reply:
If this new option is supposed to represent deferring to some other physics, I think we should come up with a different name. I think something as verbose as `defer_to_other_physics` is fine/good

Reply:
no_additional is the same intent as "none", the Physics does not set the preconditioning.
"none" is not accurate because there is a default preconditioning, and because the user might be passing petsc_options

Reply:
I think `defer_to_other_physics` is a better enumerator name

Reply:
done